lib/rules/accessor-pairs.js

```diff
@@ -63,3 +63,11 @@ module.exports = function(context) {
     };
 
 };
+
+module.exports.documentation = {
+    errors: ["Getter is not present", "Setter is not present"],
+    description: "enforces getter/setter pairs in objects",
+    ruleType: "Best Practices",
+    recommended: false,
+    default: [{getWithoutSet: false, setWithoutGet: true}]
```

Comment:
The other way is to require this as part of the schema. I really think it would help a lot of users to clearly expose defaults for all rules. Right now, not all rules document this in a easy to find way.